apps/angular/crud/src/app/models/todo-state.model.ts

```diff
@@ -0,0 +1,5 @@
+import { ITodo } from './todo.model';
+
+export interface ITodoState {
```

Comment:
don't add the letter I in front of your interface, bad practise. We already know that it's a type.